opennms-base-assembly/src/main/resources/share/mibs/opennms.mib

```diff
@@ -405,7 +405,7 @@ synchronizing		OBJECT-TYPE
 	::= { eventsAlarms 2 }
 
 -- Trap definitions
-
+-- TODO MVR can this be removed`
 eventTrap		NOTIFICATION-TYPE
```

Comment:
Can I remove this mib entry? I am unsure. Need guidance. @indigo423 @mhuot

Reply:
I removed it for now. Feel free to leave any comment, so I can address it afterwards.